pyproject.toml

```
@@ -1,3 +1,44 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "it-depends"
+version = "0.1.3"
+description = "A software dependency analyzer"
+readme = "README.md"
+license = {text = "LGPL-3.0-or-later"}
+authors = [
+    {name = "Trail of Bits"}
+]
+requires-python = ">=3.7"
```

We only test against Python 3.9+ now, so I think it's safe to make this higher.
```suggestion
requires-python = ">=3.9"
```

---

Done 👍